### PR TITLE
Add test create Atop connector

### LINKED ISSUE #15419: Add test create Atop connector
Implement test to eliminate the gap:  https://github.com/trinodb/trino/blob/5b370858ec4a9dcdf86e70de5bfce2847ca10641/plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java#L30

### PR DESCRIPTION
**Pull request by mx123**
## Description
Add test create Atop connector

Fixes #15419

### CODE REVIEW

File: plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java [hunk 1]
```diff
@@ -27,5 +34,14 @@ public void testGetConnectorFactory()
         assertInstanceOf(getOnlyElement(plugin.getConnectorFactories()), AtopConnectorFactory.class);
     }
 
-    // TODO test factory
+    @Test
+    public void testCreateConnector()
```
Comment (ebyhr) [inline plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java:38]: We can rename testGetConnectorFactory to `testCreateConnector` and reuse it instead.

File: plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java [hunk 2]
```diff
@@ -27,5 +34,14 @@ public void testGetConnectorFactory()
...
+            throws Exception
+    {
+        Plugin plugin = new AtopPlugin();
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+
+        Path atopExecutable = Files.createTempFile(null, null);
+        factory.create("test", ImmutableMap.of( "atop.executable-path", atopExecutable.toString()), new TestingConnectorContext());
```
Comment (ebyhr) [inline plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java:45]: Could you call append `.shutdown()` as other `Test*Plugin`?

**Review by ebyhr — APPROVED** [review]

File: plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java (hunk 2 above)
Comment (ebyhr) [reply plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java:45]: Also, please fix checkstyle failure:
```
Error:  src/test/java/io/trino/plugin/atop/TestAtopPlugin.java:[45,47] (whitespace) ParenPad: '(' is followed by whitespace.
```

Comment (mx123) [reply plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java]: fixed.

File: plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java (hunk 1 above)
Comment (mx123) [reply plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java:31]: renamed

**Review by findinpath — APPROVED** [review]

**Pull request merged**